src/LoggerSpy.php

```diff
@@ -44,17 +40,4 @@ public function assertNoLoggingCallsWhereMade() {
 		}
 	}
 
-	/**
-	 * @since 1.2
-	 * @return string[]
-	 */
-	public function getLogMessages(): array {
```

Comment:
I know this was never released, but this method could stay as a shortcut. Not sure if this is helpful, could also be confusing to have two ways to do the same.